src/components/faq/Question.css

```diff
@@ -30,15 +31,14 @@
     background: rgba(255, 255, 255, 0.7);
     box-shadow: 0px 4px 4px rgba(0, 0, 0, 0.25);
     border-radius: 0px 0px 10px 10px;
-    width:578px;
-    transition: height .5s ease-out;
-    height:0;
-     
+    max-width:450px;
+    transition: max-height 1s ease-in-out;
```

Comment:
The transition on collapse is a bit jarring still, it feels like a very long delay between pressing the arrow and the collapse actually triggering.  The animation of it looks good though.  Also maybe could add some transition effects to the arrow switching directions.